flink-python/apache-flink-libraries/setup.py

```diff
@@ -119,6 +119,8 @@ def find_file_path(pattern):
             find_file_path(os.path.join(OPT_PATH, "flink-python_*.jar")))
         OPT_SQL_CLIENT_JAR_NAME = os.path.basename(
             find_file_path(os.path.join(OPT_PATH, "flink-sql-client_*.jar")))
+        OPT_FLINK_SQL_CONNECTOR_HIVE_JAR_NAME = os.path.basename(
+            find_file_path(os.path.join(OPT_PATH, "flink-sql-connector-hive_*.jar")))
```

Comment:
Given Flink supports multiple Hive versions, would be good to pin to the specific version.

Reply:
Changed 👍